substratevm/DEBUGINFO.md

```diff
@@ -0,0 +1,127 @@
+Using the ptototype debug info feature
+--------------------------------------
+
+To add debug info to a generated native image add flag
+-H:+TrackNodeSourcePosition to the native image command line.
+
+  mx native-image -H:+TrackNodeSourcePosition Hello.java
```

Comment:
Nah ... instead add HostedOption GenerateDebugInfo and whenever it is > 0 make sure TrackNodeSourcePosition gets set.

Reply:
Yes, I wanted to do something like that but I was not sure exactly how to implement it. Having looked around at the code I think I worked out how to do it and have pushed a further commit which I believe does it correctly.

GenerateDebugInfo is now a HostedOption in SubstrateOptions. It uses an onValueUpdate override to test TrackNodeSourcePosition and force it to true if it is not set. Hope that is ok.

Reply:
Looks good now. Thanks!